### ISSUE TITLE
Options/Defaults/Separators change has no impact

### ISSUE THREAD

**Issue by Mondayerror** [DaxStudio/DaxStudio#560]
Changing the setting from "US/UK (a,b,c | 1.23)" to "Other (a;b;c | 1,23)" in the options menu has no impact on the exported csv  file. Also, the setting "resets" itself to  "US/UK (a,b,c | 1.23)" after the default setting has been changed to "Other (a;b;c | 1,23)" after relaunch of dax studio.

**Comment by dgosbell** [comment]
> Changing the setting from "US/UK (a,b,c | 1.23)" to "Other (a;b;c | 1,23)" in the options menu has no impact on the exported csv file.

No, it's not designed to alter the csv format, that setting only impacts the separator style used within DAX queries and expressions (I've just added a description to that setting to clarify this which will go out in the next release). 

The csv export uses the regional settings from your PC when deciding what separator styles to use. Are you saying you want to override this? If it does not appear to be using the regional settings from you PC make sure you are running the latest release, I recall fixing a bug in this area recently, but I cannot find the exact issue now, it may have been one where someone mentioned this in combination with some other issue

> Also, the setting "resets" itself to "US/UK (a,b,c | 1.23)" after the default setting has been changed to "Other (a;b;c | 1,23)" after relaunch of dax studio.

Yes, I found that bug recently it will be fixed in the next release
